test/integration/ledger/node/tx.js

```diff
@@ -172,12 +172,36 @@ const transactions = {
+    signedTxCborHex: '83a400818258202074b3783b7739037cabe8d6e9f2ca2ac8031c9d1a45334ae6853f7839709c5501018182583930de685e72586c4269087e282c9c7e78ba22082bce4a674977b4000e99b494d35f236093e7caed75d2b99b1e523cde935a6f4a2d276b9fb4011a3b6ba36a021a0002e55d05a1581df0b494d35f236093e7caed75d2b99b1e523cde935a6f4a2d276b9fb40100a200838258200fa60d5831ee0be1f221ff93ba0a4fdeca5a5866e47569607b04e287fc9b5ec0584018e8e82517d53b6a8638dc3f1540a1f165c680b7632aed8674d30fcf555872789dafe7a0e27ee172c544c96e1b7168bb2d7c3643ced05f2b1b54bc4c9bda020882582091e2647a4db84e47c895160b1f534d72c24c5eaf0aefb5f72dbfdea9913d96c658402db011af84cd38e4683dc89c83635e65f53f363c814df5f7bcd41b1608ced6c74828377b466e91b2ca31c34deb18f8410a4996cabd44dff131964084b2300b03825820b58d0c958e7e76f7cac5f3e696285ce955750967972af13ba7427cd91808a5605840b783611fe7e8b922a32a45bb09d836c168aac21e288f0c9fac515e6fdda64c3ee79adad56d0668cdc8937eab6d5168d3e8b42d4aa835097892212eff1ca4550c01818201828200581cf699c6400f85bdca54e44d0cad1f6141ce049a411c0d695fc30c3f738200581c4c139dd7e3a600fd1a4855b38cf8f731f39d3051791a5ede1f553d6cf6',
+    network: 'TESTNET',
+  },
+  multisig_stakeAddressDeregistrationCertificate: {
+    unsignedCborHex: '83a40081825820c961208acdd0d92d24837e904adbf044dd1f0d2a0fa1e1c2108299e896673d5a00018182583930de685e72586c4269087e282c9c7e78ba22082bce4a674977b4000e99b494d35f236093e7caed75d2b99b1e523cde935a6f4a2d276b9fb4011a3b8fb758021a0002dae1048182018201581cb494d35f236093e7caed75d2b99b1e523cde935a6f4a2d276b9fb4019f8200581c9a70dd7c77e9db9442b560a11446962e9d7c595274c587a62f8a0b618201828200581cf699c6400f85bdca54e44d0cad1f6141ce049a411c0d695fc30c3f738200581c4c139dd7e3a600fd1a4855b38cf8f731f39d3051791a5ede1f553d6cfff6',
+    hwSigningFiles: [signingFiles.multisigPayment0, signingFiles.multisigStake0, signingFiles.multisigStake1],
+    signedTxCborHex: '83a40081825820c961208acdd0d92d24837e904adbf044dd1f0d2a0fa1e1c2108299e896673d5a00018182583930de685e72586c4269087e282c9c7e78ba22082bce4a674977b4000e99b494d35f236093e7caed75d2b99b1e523cde935a6f4a2d276b9fb4011a3b8fb758021a0002dae1048182018201581cb494d35f236093e7caed75d2b99b1e523cde935a6f4a2d276b9fb401a200838258200fa60d5831ee0be1f221ff93ba0a4fdeca5a5866e47569607b04e287fc9b5ec05840dd08adfdbc5d9063df23dbe9e515cba6b01d4e846aea58fdee5a9ada32ea5cd5b631f7da3ff12afd116466fab7c002afce4114d13084e48719e51e84b6fa500682582091e2647a4db84e47c895160b1f534d72c24c5eaf0aefb5f72dbfdea9913d96c658409ce55b24e874c77410ef3ea9939bae13659260d3d95a723c977f9b2f3fe45cbb7414791a8f0313585d18328c5f9a38d4373c9f8ac679fbda60b44fd53090d306825820b58d0c958e7e76f7cac5f3e696285ce955750967972af13ba7427cd91808a5605840ac0558c3e80cf650038dd1e7e5ef04e7bd4560847c94d5762875601a44c07445e20b8cf7aa263794d999626695ed5c6fb0447254792d44fe555834a12f03110f01828200581c9a70dd7c77e9db9442b560a11446962e9d7c595274c587a62f8a0b618201828200581cf699c6400f85bdca54e44d0cad1f6141ce049a411c0d695fc30c3f738200581c4c139dd7e3a600fd1a4855b38cf8f731f39d3051791a5ede1f553d6cf6',
+    network: 'TESTNET',
```

Comment:
Have we covered multisig with minting? And minting plus a certficate? (to make sure we do not mess up the signing files along the way) Also, a tx with several multisig inputs sounds reasonable (perhaps one where a single key signs two inputs, and one input needs two keys, and one extra input not being signed at all, just included --- but this last item only makes sense if we are witnessing (not signing and completing the tx).

Reply:
Added transaction which has two multisig inputs, three outputs, contains a certificate, withdrawal and minting at the same time. Is that enough?
Also should I add tests for only witnessing some multisig transactions?